tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/profile/descartes_default_plan_profile.hpp

```diff
@@ -108,7 +108,8 @@ DescartesDefaultPlanProfile<FloatType>::DescartesDefaultPlanProfile(const tinyxm
         throw std::runtime_error("DescartesPlanProfile: EdgeCollisions: CollisionSafetyMargin is not a numeric "
                                  "values.");
 
-      tesseract_common::toNumeric<double>(coll_safety_margin_string, edge_collision_saftey_margin);
+      /** @todo Update xml*/
+      //      tesseract_common::toNumeric<double>(coll_safety_margin_string, edge_collision_saftey_margin);
```

Comment:
A todo to update xml?

Reply:
Yeah. Any chance we could either put that off until after #443 or someone else could chip in on that? Are you using the xml?

Reply:
I don't think it is currently being used, but I will look.